lua/plenary/async/async.lua

```diff
@@ -0,0 +1,150 @@
+local co = coroutine
+local uv = vim.loop
+
+local thread_loop = function(thread, callback)
```

Comment:
callback means a function, 

Should be particular when we have a coroutine.

This could be `co_callback`